src/backer/backering.py

```diff
@@ -310,6 +311,24 @@ def on_post(self, req, rep):
 
         cr = httping.parseCesrHttpRequest(req=req)
         serder = eventing.Serder(ked=cr.payload, kind=eventing.Serials.json)
+
+        backer_identifier = serder.ked["b"]
```

Comment:
missing check here that `backer_identifier` also equals the identifier of _this_ cardano-backer instance

Reply:
Not quite sure
checked
https://github.com/cardano-foundation/cardano-backer/pull/3/files#diff-b6c96165098df90d0baad51069a2132e161514b36ee90b346be24ffff6abacf0R303

[dab7a7ccc924fd29f3db5e45ddc13ff74f7358a7](https://github.com/cardano-foundation/cardano-backer/pull/3/files#diff-b6c96165098df90d0baad51069a2132e161514b36ee90b346be24ffff6abacf0R313)

Reply:
That comment is on the global watcher - this is the backer.

The check you have is checking that the backer identifier of the inception event matches the backer identifier defined in the SEAL. This is good.

What I meant is we need to make sure the backer identifier matches OUR identifier. This cardano-backer instance has an identifier, it's used to receipt things. Keripy will only receipt if it's correct BUT since `b` is an array we need to check.

Reply:
updated
ref here: c7be26f5668e2e1c938f663723d3ffc3b4eb817a

Reply:
I don't think that push fixes it. In this repository only, I need you to also iterate over the backer_identifiers and make sure one of them matches the identifier of this cardano-backer instance. It's printed in the console when we launch it

Reply:
https://github.com/cardano-foundation/cardano-backer/pull/3/files#diff-b6c96165098df90d0baad51069a2132e161514b36ee90b346be24ffff6abacf0R318

a bit confuse, I did check by comparing identifers and seal.bi,

Reply:
updated
ref here: 7aa0fde86644c09acfd81923a3881d90b900cc08